### ISSUE TITLE
Missing RWKV/rwkv-raven-14b from the model list?

### ISSUE THREAD

**Issue by PicoCreator** [ray-project/ray-llm#7]
RWKV is shown in the main readme
- https://github.com/ray-project/aviary#listing-all-available-models

but is missing in the current models page 
- https://github.com/ray-project/aviary/tree/master/models

**Comment by Yard1** [comment]
Yes, that is correct. We have had some trouble getting it to work, so it has been disabled for now (you can see `enabled: false` in the RWKV yaml). We will re-enable it once we can pinpoint the issue.

**Comment by PicoCreator** [comment]
Oh, the RWKV.yaml itself is not there (so cant see the enabled: false)

**Comment by Yard1** [comment]
My bad, we must have removed it from the repo at some point! Happy to add it back in.

**Comment by Yard1** [comment]
The problem was that using batch size > 1 resulted in all the responses but one turning into gibberish. We've used the Transformers integration. YAML attached:
[RWKV--rwkv-raven-14b.txt](https://github.com/ray-project/aviary/files/11667221/RWKV--rwkv-raven-14b.txt)
